Разметка всех страниц

RookHeaven opened the pull request:
Закончил вёрстку основной страницы и каталога.

---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2110671/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/2110671-sedona-34/2/

RookHeaven committed:
Made up a "header" in index.html

RookHeaven committed:
Made up index.html

RookHeaven committed:
Made changes in index.html and in catalog.html

1. Добавил по классу кнопкам в файле index.html
2. Сделал шапку, фильтры, копировал и вставил блок с подпиской и футер

RookHeaven committed:
Finished catalog.html

Закончил вёрстку catalog.html, прогнал в валидаторе страницы и исправил ссылку телефона в index.html и catalog.html

Komnatnaya commented on index.html:
```diff
@@ -4,9 +4,229 @@
+            <ul class="features-list">
+
+              <li class="features-item features-item-house">
+                <img src="#" alt="Иконка дома">
```
Иконки - декоративные элементы, а в тег img мы ставим только контентные изображения.

Komnatnaya commented on catalog.html:
```diff
@@ -0,0 +1,401 @@
+            <li class="breadcrumbs-item">
+
+              <a class="breadcrumbs-link" href="index.html">
+                <img class="breadcrumbs-image" src="#" alt="На главную страницу">
```
Это тоже иконка, пользователю важна сама ссылка, а не то как выглядит иконка - так что это декор.

Komnatnaya commented on catalog.html:
```diff
@@ -0,0 +1,401 @@
+          <span class="sorting-results">Найдено гостиниц: 38</span>
+
+          <select class="select-control" aria-label="Сортировка">
+            <option value="cheap">Сначала дешёвые</option>
```
Стоит добавить несколько рандомных option

Komnatnaya commented on catalog.html:
```diff
@@ -0,0 +1,401 @@
+              </p>
+
+              <a class="hotel-more-button button button-brown" href="#">Подробнее</a>
+              <button class="hotel-favourites-button button button-blue" type="submit">В избранное</button>
```
type="submit" - это кнопка отправки формы, а тут нет формы, значит, тип неверный.

Komnatnaya commented on catalog.html:
```diff
@@ -0,0 +1,401 @@
+          <div class="hotel-select">
+            <label class="select-title">Гостиниц на странице:</label>
+            <select class="select-number" aria-label="Сортировка страниц">
+              <option value="four">4</option>
```
Стоит добавить несколько рандомных option

RookHeaven committed:
Made improvements to index.html and catalog.html

Добавил точки в конце описаний, убрал иконки, хлебную крошку сделал <span> с описанием, добавил несколько дополнительных option в select-ы
